test-data/unit/check-warnings.test

```diff
@@ -141,7 +141,7 @@ from typing import Any
 def g() -> Any: pass
 def f() -> int: return g()
 [out]
-main:4: warning: Returning Any from function with declared return type "builtins.int"
+main:4: warning: Returning Any from function with declared return type "int"
```

Comment:
Maybe add one more test for situations where the special formatting takes place (like with callable types and/or long unions and/or ``NoReturn`` and/or module type)?